homeassistant/components/homekit/const.py

```diff
@@ -37,10 +38,12 @@
 CONF_STREAM_ADDRESS = "stream_address"
 CONF_STREAM_SOURCE = "stream_source"
 CONF_SUPPORT_AUDIO = "support_audio"
+CONF_VIDEO_CODEC = "video_codec"
 CONF_VIDEO_MAP = "video_map"
 CONF_VIDEO_PACKET_SIZE = "video_packet_size"
 
 # #### Config Defaults ####
+DEFAULT_AUDIO_CODEC = "libopus -application lowdelay"
```

Comment:
```suggestion
DEFAULT_AUDIO_CODEC = "libopus"
```

We should figure out the `-application lowdelay` for them and append it in the code.

Reply:
Solved.